problem_builder/answer.py

```diff
@@ -115,6 +115,20 @@ def add_error(msg):
         if not data.name:
             add_error(u"A Question ID is required.")
 
+    def student_view_user_state(self, context=None):
```

Comment:
@bradenmacdonald student_view_data does not have it. In that case either `student_view_data` is broken or my assumptions are not quite correct. Do we need `student_view_user_state` to appear in the response from course API, or just as a handler on a block?